examples/compose/vtcompose/vtcompose.go

```diff
@@ -317,10 +317,10 @@ func getTableName(sqlFile string) string {
 		log.Fatalf("reading sqlFile file %s: %s", sqlFile, err)
 	}
 
-	r, _ := regexp.Compile("CREATE TABLE ([a-z_-]*) \\(")
+	r, _ := regexp.Compile("(?i)CREATE[[:space:]]*TABLE[[:space:]]*(IF NOT EXISTS)?[[:space:]]*([a-z_-]*)[[:space:]]*\\(")
```

Comment:
Gah! I'm sorry... I just realized that we should be requiring *at least* one space in a few spots there so that we don't also match e.g. `CREATETABLE`:
```suggestion
	r, _ := regexp.Compile("(?i)CREATE[[:space:]]+TABLE[[:space:]]+(IF NOT EXISTS)?[[:space:]]+([a-z_-]*)[[:space:]]*\\(")
```
https://go.dev/play/p/nxjMA7SiTsW

I'm sorry that I caused yet another round trip on that. 😞